# Request
string data
---
# Response
bool success
